src/cyw43_ll.c

```diff
@@ -364,7 +364,7 @@ static void cyw43_write_backplane(cyw43_int_t *self, uint32_t addr, size_t size,
     cyw43_set_backplane_window(self, CHIPCOMMON_BASE_ADDRESS);
 }
 
-static int cyw43_check_valid_chipset_firmware(cyw43_int_t *self, size_t len, uintptr_t source) {
+static int cyw43_check_valid_chipset_firmware(size_t len, uintptr_t source) {
```

Comment:
I would prefer to add a line at the start of this function like `(void)self;`.  That's how this is handled elsewhere in this driver.

Reply:
done